README.md

```diff
@@ -0,0 +1,74 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Developer Job Search
+
+> Developer Job Search is a React web app for searching for a job. This app is built using reed.co.uk job search API, the UK's #1 job site. Feel free to use my web app for browsing for jobs posted on the reed.co.uk website. The design is mobile responsive so you can use it on any device. However, take note that I am using limited hosting and my private proxy server, so bandwidth may be an issue. Please do let me know when you have this issue and I will respond accordingly.
+
+> ![Developer Job Search](./screenshot.png)
+
+## Built With
+
+- React
+- Redux
+- JavaScript (ES6+)
+- [Reed job search API](https://www.reed.co.uk/developers)
+- Jest
+- React testing library
+
+## Live Demo
+
+:fire: https://wondrous-creponne-38ed9a.netlify.app/ :fire:
+
+## Video Demo
+
+https://www.loom.com/share/be2634dc593b4ec9aa094e869949b3a0
+
```

Comment:
- [ ] Kindly look into your demo presentation. When I visit the link all I see is a blank screen and a moving mouse. Kindly fix that. I've tried with different browsers including incognitos - same result

  ![rirop](https://user-images.githubusercontent.com/54848363/204513979-b3fc4334-fb2a-493d-9f69-79f60b1ee25f.jpg)